worker/include/UnixSocketSfuCpp.hpp

```diff
@@ -0,0 +1,12 @@
+ChannelReadFreeFn channelReadFn (uint8_t** message,
```

Comment:
maybe there is no need for an additional hpp file since it is only used in one place
(can be declared with extern)